src/Compilers/Core/Portable/Symbols/INamedTypeSymbol.cs

```diff
@@ -65,6 +65,13 @@ public interface INamedTypeSymbol : ITypeSymbol
         ImmutableArray<ITypeSymbol> TypeArguments { get; }
 
         /// <summary>
+        /// Returns custom modifiers for the type argument that has been substituted for the type parameter. 
+        /// The modifiers correspond to the type argument at the same ordinal within the <see cref="TypeArguments"/>
+        /// array.
+        /// </summary>
```

Comment:
Maybe mention that it returns empty arrays (not default) when no custom modifiers.